guide/14-deep-learning/text-classification-workflow.ipynb

```diff
@@ -0,0 +1,1239 @@
+{
```

Comment:
<p>The TOC can be restructured as per below suggestions:</p><p> </p><ul><li>The model uses transformer architecture can be emphasized in <a href="https://app.reviewnb.com/Esri/arcgis-python-api/pull/798/files/#Introduction" target="_blank">Introduction</a> and <a href="https://app.reviewnb.com/Esri/arcgis-python-api/pull/798/files/#Transformer-Basics" target="_blank">Transformer Basics</a> can be made a top level heading.</li><li><a href="https://app.reviewnb.com/Esri/arcgis-python-api/pull/798/files/#Prerequisites" target="_blank">Prerequisites</a> can be the software and data requirements.  </li><li><a href="https://app.reviewnb.com/Esri/arcgis-python-api/pull/798/files/#How-to-choose-an-appropriate-model-for-your-dataset?" target="_blank">How to choose an appropriate model for your dataset?</a> can be a sub under TextClassifier model heading.</li><li><a href="https://app.reviewnb.com/Esri/arcgis-python-api/pull/798/files/#Supported-formats-for-labeled-training-data" target="_blank">Supported formats for labeled training data</a> can be moved under data preparation.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/798/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Done the necessary changes,

Reply:
<p>Few typo corrections on the sample NB suggested in the below doc:</p><p> </p><p>https://esriis.sharepoint.com/:w:/s/EsriRDCenterNewDelhi/EeebPFZsfPhCs2yOcB6CiKUBzxb-LdNdaH-4h0PjUKb0Lw?e=LsmbF6</p>

Reply:
<p>Done the necessary changes</p>